integration/containerd/runk/runk-tests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2021-2022 Kata Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will validate runk with containerd
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+dir_path=$(dirname "$0")
+source /etc/os-release || source /usr/lib/os-release
+source "${dir_path}/../../../lib/common.bash"
+source "${dir_path}/../../../.ci/lib.sh"
+
+KATACONTAINERS_REPO=${katacontainers_repo:="github.com/kata-containers/kata-containers"}
+RUNK_SRC_PATH="${GOPATH}/src/${KATACONTAINERS_REPO}/src/tools/runk"
+RUNK_BIN_PATH="/usr/local/bin/runk"
+TEST_IMAGE="docker.io/library/busybox:latest"
+CONTAINER_ID="id1"
+PID_FILE="${CONTAINER_ID}.pid"
+
+setup() {
+    # can't find cargo when make runk below, so we install rust to make runk build correctly
+    "${dir_path}/../../../.ci/install_rust.sh" && source "$HOME/.cargo/env"
+    echo "restart containerd service"
+    sudo systemctl restart containerd
+    echo "pull container image"
+    sudo ctr images pull ${TEST_IMAGE}
+}
+
+install_runk() {
+    echo "Install runk"
+    pushd ${RUNK_SRC_PATH}
+    make
+    sudo make install
+    popd
+}
+
+test_runk() {
+    echo "start container with runk"
+    sudo ctr run --pid-file ${PID_FILE} --rm -d --runc-binary ${RUNK_BIN_PATH} ${TEST_IMAGE} ${CONTAINER_ID}
```

Comment:
any reason why you are putting runc (--runc-binary) and not kata? or why not io.containerd.runc.v2?

Reply:
Just like runc, [runk](https://github.com/kata-containers/kata-containers/tree/main/src/tools/runk) is a standard OCI container runtime. It is in development and lacks testing. So I let containerd use runk to verify its functionality.